src/reporters/logging_reporter.js

```diff
@@ -30,7 +30,7 @@ export default class LoggingReporter {
     }
 
     report(span: Span): void {
-        this._logger.info(`Reporting span ${JSON.stringify(span)}`);
+        this._logger.info(`Reporting span ${span.context().toString()}`);
```

Comment:
Interpolated values are automatically converted to strings:

```javascript
`${{toString: () => 'abc'}}`
// -> "abc"
```

Reply:
I don't follow. IIRC Javascript does not have Java's convention of automatically calling `toString()` method when converting to strings.

Reply:
JS does call `toString()` when converting to string.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/toString

>Every object has a toString() method that is automatically called when the object is to be represented as a text value or when an object is referred to in a manner in which a string is expected.